compiler-utils/src/main/java/com/squareup/anvil/compiler/internal/reference/AnnotationReference.kt

```diff
@@ -27,6 +32,8 @@ public sealed class AnnotationReference {
   public val fqName: FqName get() = classReference.fqName
   public val module: AnvilModuleDescriptor get() = classReference.module
 
+  public abstract fun boundTypeOrNull(): FqName?
```

Comment:
I'm sorry, but I didn't catch this the first time. This method should now return `ClassReference?`.